### DISCUSSION TITLE
Possibility to remove "selector" from generated classes?

### DISCUSSION THREAD

**Discussion by dunklesToast** [mantinedev/mantine#4431] [Q&A]
Hey,

we're currently searching for steps to reduce our bandwidth usage and wondered if it's possible to remove the static selector parts of the classNames and just keep the 7 character long id's that are also generated if you use `sx`. Kinda like you could do with [localIdentName](https://webpack.js.org/loaders/css-loader/#localidentname) in CSS Loader.

![image](https://github.com/mantinedev/mantine/assets/17279485/724f3d4d-75c9-46e0-92c9-1cd589ffaa34)

While we know that this will have a very small impact on the bandwidth used, we're just interested in if this is possible so any feedback would be appreciated :)

**Comment by rtivital** [comment]
No, currently it is not supported
